man/mblaze.7

```diff
@@ -87,92 +87,122 @@ OpenSMTPD,
 Postfix,
 or similar.
 .Nm
-expects your mail to reside in Maildir folders.
+expects your messages to reside in maildir folders.
 .Pp
 .Nm
-operates directly on Maildir folders and doesn't
+operates directly on maildir folders and doesn't
 use its own caches or databases.
 There is no setup needed for many uses.
-All tools have been written with performance in mind.
-Enumeration of all mails in a Maildir is avoided unless necessary,
+All utilities have been written with performance in mind.
+Enumeration of all messages in a maildir is avoided unless necessary,
 and then optimized to limit syscalls.
-Parsing mail metadata is optimized to limit I/O requests.
-Initial operations on a large Maildir may feel slow, but as soon as they
+Parsing message metadata is optimized to limit I/O requests.
+Initial operations on a large maildir may feel slow, but as soon as they
 are in the file system cache, everything is blazingly fast.
-The tools are written to be memory efficient (i.e. not wasteful), but
-whole messages are assumed to fit into RAM easily (one at a time).
+The utilities are written to be memory efficient
+.Po
```

Comment:
Not sure if the `.Po`/`.Pc` is nicer here (as we don't use macros inside parens), but for single line usage one better use `.Pp` I think.

Reply:
Not sure what you mean here. How does `.Pp` come into play (specifically in relation to `.Po` / `.Pc`? Ah - do you mean `. Pq` for single line use??

Reply:
I meant `.Pq`, but I'm not sure it works well.

Reply:
Done, with `.Pq`.